### DISCUSSION TITLE
How to create UI for threaded messages?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#13205] [General - Components / Directives / etc] [answered]
I know Quasar has a [Chat Message](https://quasar.dev/vue-components/chat#introduction) UI. But that is built for instant mesages.

Instead I want to create something similar to support tickets, where it shows a message summary and then you click to read the full message text. As an example here is an image from Zoho Desk (support ticket software) where the each message is "collapsed" except the last one which I have clicked to show it expanded: https://imgur.com/a/kWd16q6

Another way to describe this type of UI would be the Gmail email client. When you view threaded email messages you can click on each message to show the full text.

Is it possible to create something like this using Quasar?

**Comment by MilosPaunovic** [comment]
You can utilize `QTree` for those purposes.
[https://quasar.dev/vue-components/tree](https://quasar.dev/vue-components/tree)

**Comment by TobyMosque** [comment, marked as answer]
IMHO that is a [QList](https://quasar.dev/vue-components/list-and-list-items) with [QExpansionItem](https://quasar.dev/vue-components/expansion-item#introduction)

**Reply by BenJackGill** [reply]
Oooh this looks like a promising combo.

`QList` docs even has an [Emails example](https://quasar.dev/vue-components/list-and-list-items#example--emails) to give me a head start, but without the expansion part.
